Docs/architecture.md

```diff
@@ -15,8 +15,4 @@ However, customers looking for any of the following are expected to prefer a set
   - Gateway management.
 - Homogenous management of devices and gateways independent of connectivity technology.
 
-## High Level Architecture
-Below is a high level diagram of how the framework works, for more details refer to the [dev guide](devguide.md).
```

Comment:
To make it easier to read, split the sentence like this: 
Below is a high level diagram of how the framework works. For more details refer to the [dev guide](devguide.md).